templates/page.html

```diff
@@ -26,13 +26,13 @@ <h1 itemprop="headline">{{ page.title }}</h1>
                 {% endif %}
                 {% if page.taxonomies.categories %}
                     {% set category = page.taxonomies.categories[0] %}
-                    in <a href="{{ get_taxonomy_url(kind="categories", name=category) | safe }}">{{ category }}</a>
+                    in <a href="{{ get_taxonomy_url(kind="categories", name=category, lang=page.lang) | safe }}">{{ category }}</a>
```

Comment:
This seems to be why the CI is failing. Is there any reason why a page wouldn't have a `lang` field?

Reply:
No, it should always have it

Reply:
is there a way to download the CI so I can try to debug it locally?

Reply:
Not really, but the CI just builds the site so `zola build` should be equivalent

Reply:
hmm, thats strange. I tried it on both of my machines, and it built with no issues on both. maybe it's something to do with my rust toolchain or something I have installed?

Reply:
Definitely change the CI version, 0.5 is ancient

Reply:
ok that should be fixed, i'll set up netlify.toml files for all the themes this weekend so it's easier to keep track